Sources/SwiftFormatRules/DoNotUseSemicolons.swift

```diff
@@ -77,3 +98,12 @@ extension Diagnostic.Message {
   static let removeSemicolonAndMove = Diagnostic.Message(
     .warning, "remove ';' and move the next statement to a new line")
 }
+
+/// Protocol that declares support for accessing and modifying a token that represents a semicolon.
```

Comment:
Can you toss this (and the extensions below) in a separate file (SemicolonSyntax.swift)? We have various extensions/convenience things like this that we should look at someday contributing back to swift-syntax, so keeping it separate from the rule will make sure that we don't forget about it as easily.

Reply:
Done.